lib/ssl/src/ssl_connection.erl

```diff
@@ -1179,6 +1179,34 @@ certify(internal, #client_key_exchange{exchange_keys = Keys},
 	#alert{} = Alert ->
 	    handle_own_alert(Alert, Version, ?FUNCTION_NAME, State)
     end;
+%% The response will be handled only when a certificate status request has
+%% been sent by the client and confirmed by the server.
+certify(internal, #certificate_status{response = OcspRespDer},
+        #state{handshake_env = #handshake_env{
+                   ocsp_stapling_state = #{
+                       is_client_hello_requested := true,
+                       is_server_hello_requested := true,
+                       ocsp_responder_certs := ResponderCerts,
```

Comment:
ocsp_reponder_certs should probably go to static_env. Should we put these certs into cert_db?

```erlang
-record(static_env, {
                     role                  :: client | server,
                     transport_cb          :: atom(),   % callback module
                     protocol_cb           :: tls_connection | dtls_connection,
                     data_tag              :: atom(),   % ex tcp.
                     close_tag             :: atom(),   % ex tcp_closed
                     error_tag             :: atom(),   % ex tcp_error
                     passive_tag           :: atom(),   % ex tcp_passive
                     host                  :: string() | inet:ip_address(),
                     port                  :: integer(),
                     socket                :: port() | tuple(), %% TODO: dtls socket
                     cert_db               :: reference() | 'undefined',
                     session_cache         :: db_handle(),
                     session_cache_cb      :: atom(),
                     crl_db                :: term(),
                     file_ref_db          :: db_handle(),
                     cert_db_ref          :: certdb_ref() | 'undefined',
                     trackers              :: [{atom(), pid()}] | 'undefined' %% Trac
or listen socket
                    }).
```